docs/en/stack/ml/df-analytics/ml-supervised-workflow.asciidoc

```diff
@@ -0,0 +1,139 @@
+a numerical value? The answers help you choose the type of analysis that fits 
+your use case.
+
+After you identify the problem, consider which of the {ml-features} are most 
+likely to help you solve it. Unsupervised learning – like {anomaly-detect} or 
+{oldetection} – does not require a labelled data set to train the model on while 
+supervised learning requires a data set that contains the known values that the 
+model can be trained on.
+
+{stack} provides the following types of supervised learning: 
+
+* {regression}: predicts **continuous, numerical values** like the response time 
+  of a web request. 
+* {classification}: predicts **discrete, categorical values** like whether a 
+  https://www.elastic.co/blog/machine-learning-in-cybersecurity-training-supervised-models-to-detect-dga-activity[DNS request originates from a malicious or benign domain]. 
+
+
+[[prepare-transform-data]]
+== Prepare and transform data
+
+You have defined the problem and selected an appropriate type of analysis. The 
+next step is to produce a high-quality data set in {es} with a clear 
+relationship to your training objectives. If your data is not already in {es}, 
+this is the stage where you develop your data pipeline. If you want to learn 
+more about how to ingest data into {es}, refer to the 
+{ref}/ingest.html[documentation].
+
+{regression-cap} and {classification} are supervised {ml} techniques, therefore 
+you must supply a labelled data set for training. This is often called the 
+"ground truth". The "ground truth" allows the training process to identify 
+relationships among the various characteristics of the data and the predicted 
+value as well as takes a critical role in model evaluation.
+
+An important requirement is a data set that is large enough to train a model. 
+For example, if you would like to train a {classification} model that decides 
+whether an email is a spam or not, you need a labelled data set that contains 
+enough data points from each possible category to train the model. What counts 
+as "enough" depends on various factors like the complexity of the problem or 
+the {ml} solution you have chosen. There is no exact number that fits every 
+use case; deciding how much data is acceptable is rather a heuristic process 
+that might involve iterative trials.
+
+Before you train the model, consider preprocessing the data. In practice, the 
+type of preprocessing depends on the nature of the data set. Preprocessing can 
+include, but is not limited to, mitigating redundancy, reducing biases, applying 
+standards and/or conventions, data normalization, and so on.
+
+{regression-cap} and {classification} requires specifically structured source 
+data: a two dimensional tabular data structure. For this reason, you might need 
+to {ref}/transforms.html[{transform}] your data to create a {dataframe} which 
+can be used as the source for {dfanalytics} like {regression} and 
```

Comment:
Since the paragraph started with "regression and classification", I think they're redundant here:
```suggestion
can be used as the source for these types of {dfanalytics}.
```